templates/leaderboard/leaderboard.html

```diff
@@ -9,24 +9,76 @@
 {% endblock %}
 
 {% block content %}
-    <h2>Investments</h2>
-    <div class="break">
-        <hr>
-    </div>
     <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_raised %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_invested %}
-    </div>
-    <h2>Earnings</h2>
-    <div class="break">
-        <hr>
-    </div>
-    <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_earned_artists %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+        {% include "leaderboard/includes/list.html" %}
     </div>
 {% endblock %}
 
 {% block extrajs %}
     <script type="text/javascript" src="{% static "js/vendor/Chart.min.js" %}"></script>
+
+    <script>
+        var ctx = document.getElementById("InvestorChart").getContext('2d');
+        var myChart = new Chart(ctx, {
+            type: 'bar',
+            data: {
+                labels: [
+                    {% if top_earned_investors %}
+                        {% for top_earned_investors_context in top_earned_investors %}
+                            "{{ top_earned_investors_context.name }}"{% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    {% endif %}],
+                datasets: [{
+                    label: 'Amount Invested',
+                    data: [
+                        {% if top_invested %}
+                            {% for top_invested_context in top_raised %}
+                                {{ top_invested_context.amount|notrail_floatformat:2 }}{% if not forloop.last %},{% endif %}
+                            {% endfor %}
+                        {% endif %}],
+                    backgroundColor: "#85929E"
+                }, {
+                    label: 'Amount Earned',
+                    data: [
+                        {% if top_earned_investors %}
+                            {% for top_earned_investors_context in top_earned_investors %}
+                                {{ top_earned_investors_context.amount|notrail_floatformat:2 }}{% if not forloop.last %},{% endif %}
+                            {% endfor %}
+                        {% endif %}],
+                    backgroundColor: "#4cbd7e"
+                }]
+            }
+        });
+        var ctx = document.getElementById("ArtistChart").getContext('2d');
```

Comment:
`ctx` and `myChart` should probably be named something else, since the variable collides with the definition earlier. This might not work on every browser.